playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_shadow.html.erb

```diff
@@ -0,0 +1,82 @@
+                <% end %>
+             
+                <%= pb_rails("body", props: {classname: "flex-item", padding_right: "sm", margin_left: "sm" }) do %>
+                  <%= pb_rails("caption", props: { text: "value: 0 4px 10px 0 rgb(60 106 172 / 16%)", size: "xs", padding_right: "xs" }) %>
+                  <% end %>
+            <% end %>
+            <% end %>
+            
+            <br>
+            
+            <%= pb_rails("card", props: {shadow: "deeper", padding: "none"}) do %>
+              <%= pb_rails("flex", props: { wrap: true, justify: "between", align: "center" }) do %>
+                <%= pb_rails("body", props: {classname: "flex-item", margin_right: "md" }) do %>
+                  <%= pb_rails("flex", props: { wrap: true, align: "center" }) do %>
+                    <%= pb_rails("circle_icon_button", props: { variant: "link", icon: "copy", padding_right: "xs" }) %>
+                    $shadow_deeper
+                    <% end %>
+                <% end %>
+             
+                <%= pb_rails("body", props: {classname: "flex-item", padding_right: "sm", margin_left: "sm" }) do %>
+                  <%= pb_rails("caption", props: { text: "value: 0 12px 28px 0 rgb(60 106 172 / 18%)", size: "xs", padding_right: "xs" }) %>
+                  <% end %>
+            <% end %>
+            <% end %>
+            
+            <br>
+
+            <%= pb_rails("card", props: {shadow: "deepest", padding: "none"}) do %>
+              <%= pb_rails("flex", props: { wrap: true, justify: "between", align: "center" }) do %>
+                <%= pb_rails("body", props: {classname: "flex-item", margin_right: "md" }) do %>
+                  <%= pb_rails("flex", props: { wrap: true, align: "center" }) do %>
+                    <%= pb_rails("circle_icon_button", props: { variant: "link", icon: "copy", padding_right: "xs" }) %>
+                    $shadow_deepest
+                    <% end %>
+                <% end %>
+             
+                <%= pb_rails("body", props: {classname: "flex-item", padding_right: "sm", margin_left: "sm" }) do %>
+                  <%= pb_rails("caption", props: { text: "value: 0 30px 38px 4px rgb(60 106 172 / 20%), 0 2px 14px 4px rgb(60 106 172 / 10%)", size: "xs", padding_right: "xs" }) %>
+                  <% end %>
+            <% end %>
+            <% end %>
+            <% end %>
+            
+            <br>
+           
+      <% react_contents = Rails.root.join("app/views/pages/code_snippets/shadow_erb.txt").read %>
+    
+      <%= pb_rails("background", props: { background_color: "dark", padding: "xs", id: "code-wrapper" }) do %>
+        <%= pb_rails("caption", props: { padding_x: "md", padding_top: "md", text: "Example in Sass" }) %>
+          <div class="pb--codeCopy" data-action="toggle" data-togglable="code_example">
+            <pre class="highlight"><%= raw rouge(react_contents, "react") %></pre>
+          </div>        
+      <% end %>
```

Comment:
I'd like to move this variable to the controller. See this example: https://github.com/powerhome/playbook/blob/master/playbook-website/app/controllers/pages_controller.rb#L85

Also, note the subtle changes in this code example. You might consider copying this: https://github.com/powerhome/playbook/blob/master/playbook-website/app/views/pages/visual_guidelines_partials/_border_radius.html.erb#L49-L54